docs-app/app/components/widgets/widget-3/tour-schedule/responsive-image.gts

```diff
@@ -1,7 +1,8 @@
 import { action } from '@ember/object';
 import Component from '@glimmer/component';
 import { tracked } from '@glimmer/tracking';
-import { containerQuery, type Dimensions } from 'ember-container-query';
```

Comment:
**TODO**

Reintroduce this line after releasing `ember-container-query@v4.0.0` (on May 1 or after).